src/client/app/redux/slices/adminSlice.ts

```diff
@@ -59,6 +62,10 @@ export const adminSlice = createSlice({
 			state.defaultAreaUnit = action.payload;
 			state.submitted = false;
 		},
+		updateDefaultTemperatureUnit: (state, action: PayloadAction<TemperatureUnitType>) => {
```

Comment:
While looking over the new code I notice that many (maybe all but did not check) the updates for preferences are not used and some are not exported. Thus, I'm wondering if they are needed or if I'm overlooking something. I'm wondering if they are left from when admin pages updated global state before sending to the server on save. Now it is all done in local state. I did not check carefully but I think preferences are all updated as a group on the server and the in Redux state. This is only partly about this PR but if you have thoughts that would be nice. If they are not needed then OED can open an issue to remove them.